conf/unloader.cfg

```diff
@@ -13,8 +13,12 @@ dir_location=/var/spool/apel/
 # VStarRecords
 table_name = VJobRecords
 
-# Only for job records and summaries
+# Only used for job and storage accounting.
```

Comment:
`send_ur` MUST be set to true for storage, right? That should be made clear, otherwise this change doesn't address your main point the the PR description.

Reply:
how about I change that bit to something like
```
# For job accounting, if send_ur is true, records and summaries will be sent in the OGF Usage Record format.
# For storage accounting, send_ur must be true.
```

Reply:
Yes, that's better. Just change "job" to "grid".